docs/sdks/sdk-ref-android.md

```diff
@@ -36,9 +36,17 @@ The SDK for Android is designed to generate and/or manage UID2 identity on behal
 
 By default, the SDK automatically refreshes UID2 identity based on expiration dates. However, you can disable this to allow implementing apps to manage the UID2 identity lifecycle manually.
 
+## UID2 Account Setup
+
+To integrate with UID2, you'll need to have a UID2 account. If you haven't yet created an account, first follow the steps described on the [Account Setup](../getting-started/gs-account-setup.md) page.
+
 ## API Permissions
```

Comment:
i don't think we need an explicit _API permissions_ section any more so can fold the rest under _UID2 Account Setup_

Do you wanna mention like on other pages:

The steps you'll take in the UID2 Portal are different depending on whether your implementation will be client-side, client-server, or server-side. Specific instructions are in each implementation guide. (and link to  list of mobile integration options on http://localhost:3006/docs/guides/integration-mobile-overview#client-side-or-client-server-integration )